entrypoint.sh

```diff
@@ -78,8 +90,9 @@ log_invalid_files(){
     for file in ${invalid_files[@]}; do
         printf "\n\t\e[33m %s \e[0m\n" "$file"
     done
-    printf "\n\e[33m ShellCheck only supports sh/bash/dash/ksh scripts. For supported scripts to be scanned, make sure to add a proper shebang on the first line of the script.\e[0m\n"
+    printf "\n\e[33m ShellCheck only supports sh/bash/dash/ksh scripts. For supported scripts to be scanned, make sure to add a proper shebang on the first line of the script.\n\n To fix the warning for the unsupported scripts, use the 'excluded_paths' input. For more information check:
```

Comment:
Regarding `To fix the warning for the unsupported scripts, use the 'excluded_paths' input`, should we also indicate that they can ignore files they don't want to lint (which could already a supported script). Something like `To ignore specific files or unsupported scripts, use the ...`?